### DISCUSSION TITLE
Whether whereabouts works without any CNI plugin manager like multus

### DISCUSSION THREAD

**Discussion by adhithacholank** [k8snetworkplumbingwg/whereabouts#174] [Q&A] [answered]
Whether whereabouts works without any CNI plugin managers like multus... whereabouts is working fine with multus.

**Comment by maiqueb** [comment, marked as answer]
I am not a maintainer, but from my understanding, whereabouts itself **does not** require multus. 

... however, the `ip-reconciler` tool shipped in multus does rely on the `network-status` annotations, and stays away from the default cluster network, so it can be said that it is only useful for any meta-cni plugin that follows the [Kubernetes Network Custom Resource Definition De-facto Standard](https://github.com/k8snetworkplumbingwg/multi-net-spec/blob/master/v1.1/%5Bv1.1%5D%20Kubernetes%20Network%20Custom%20Resource%20Definition%20De-facto%20Standard.pdf). This particular attribute is described on section 5.

**Reply by dougbtv** [reply]
Just giving a +1. The primary Whereabouts tool doesn't explicitly require Multus. 

Miguel's correct however, that without reconciliation, you may be in a situation where you have stale IP addresses that can't be automatically cleaned out, rendering Whereabouts less useful.

Overall, I'd likely agree that it's most useful in tandem with a CNI delegating plugin that follows the Network Plumbing Working Group's de-facto standard.
